src/app/custom-navigation-mode/custom-navigation-mode.component.html

```diff
@@ -0,0 +1,88 @@
+<p>Both wizards below use a custom navigation mode that allows the user to return to any steps he/she already visited in any direction.  This page illustrates several options to configure a wizard with a custom navigation mode.</p>
+
+<p>Option 1:</p>
+<ol>
+  <li>Create a custom navigation mode factory.  Most likely you will need to extend the <code>BaseNavigationModeFactory</code> class provided by the library.  Alternatively, in the rare occassion when you need to implement a custom navigation mode factory completely from scratch, implement the <code>NavigationModeFactory</code> interface.</li>
+  <li>In your custom navigation mode factory, override <code>createByName</code> to support a custom navigation mode name, e.g. 'custom'.</li>
+  <li>In your application module, import <code>ArchwizardModule</code> as <code>ArchwizardModule.forRoot({{'{'}} navigationModeFactory: ... })</code>, passing in the <code>navigationModeFactory</code> field an instance of your custom navigation factory.</li>
+  <li>Configure a wizard component to use you custom navigation mode with <code>&lt;aw-wizard navigationModeInput="custom"&gt;</code>.</li>
```

Comment:
... to use you**r** custom navigation mode ...

Reply:
Thanks! Fixed in e05d500